packages/edit-site/src/components/global-styles/screen-revisions/style.scss

```diff
@@ -13,8 +13,6 @@
 
 .edit-site-global-styles-screen-revisions__revision-item {
 	position: relative;
-	padding-left: $grid-unit-20;
-	overflow: hidden;
```

Comment:
I had to turn off `overflow:hidden` in order to display the focus outline correctly. From what I've tested, there doesn't seem to be any problem with removing this style, but if you have any problems please let me know.